core/config.class.inc.php

```diff
@@ -1399,6 +1399,14 @@ class Config
 			'source_of_value' => '',
 			'show_in_conf_sample' => true,
 		],
+		'theme_precompilation_enabled' => [
```

Comment:
```suggestion
		'theme.enable_precompilation' => [
```

Do you mind if we "namespace" this param (and the future ones) so it will be easier to find / documentation parameters related to themes? It's what we tried to do on the new 3.0 parameters (eg. quick_create.xxx, activity_panel.xxx, ...)